multimedia/icecast/files/icecast.init

```diff
@@ -1,12 +1,21 @@
 #!/bin/sh /etc/rc.common
-# Example script
-# Copyright (C) 2014 OpenWrt.org
+
+# Startup script
+# Copyright (C) 2015 OpenWrt.org
  
 USE_PROCD=1
-START=99
+START=90
 STOP=15
 
 start_service() {
+	user_exists icecast 87 || user_add icecast 87
+	group_exists icecast 87 || group_add icecast 87
```

Comment:
Are these two lines actually necessary? Placing `USERID:=...` should do the job; see [here](https://wiki.openwrt.org/doc/devel/packages). Can you check if user+group are created without these lines? Thx!